VoxelisedSimulation/SubScripts/ComputePoissonDataCorrections.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+## AUTHOR: Robert Twyman
+## AUTHOR: Kris Thielemans
+## Copyright (C) 2021 University College London
```

Comment:
presumably you copied lines from previous scripts, so the date should reflect that. start from 2020?